07_errors/jimbotech/sync_mapstore.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"math/rand"
+	"sync"
+)
+
+// SyncMapStore stores puppies threadsafe.
+type SyncMapStore struct {
+	mux sync.Mutex
```

Comment:
`mux` is usually the name for a multiplexer, not a mutex. You see that name used a lot for `http.ServeMux`. I think `m` is fine as a name, as it is always used with `Lock()` or `Unlock()` after it - it is clear from usage that `m` is a mutex. Otherwise be verbose and call it `mutex`.

Reply:
Done